### PR TITLE
Added a new job tab for Specific station jobs

### LINKED ISSUE #15500: we should move station specific jobs to their own tab in role select
less confusion, say what stations theyre available on, etc

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
## About the PR

Closes #15500
image

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Vasilis
- add: Added a new job tab for Station specific jobs, as well as saying what stations have the role.

### CODE REVIEW

**Comment by brainfood1183** [comment]
I personally dont like these jobs being station specific and would instead prefer if each station could have up to 2 of these jobs and no more than 2. Any combination of the 2 jobs. This is so the roles get used a little more and are not obsolete when a specific map drops out of rotation or from lack of maintenance. Just my opinion though.

**Comment by VasilisThePikachu** [comment]
> I personally dont like these jobs being station specific and would instead prefer if each station could have up to 2 of these jobs and no more than 2. Any combination of them. This is so the roles get used a little more and are not obsolete when a specific map drops out of rotation or from lack of maintenance. Just my opinion though.

Maybe this will encourage mappers to put them in their own maps, for now saying what station they are available on is a nice idea.

**Comment by Emisse** [comment]
we can discuss this in cartography brainfood

File: Resources/Locale/en-US/job/department-desc.ftl [hunk 1]
```diff
@@ -5,3 +5,4 @@ department-Engineering-description = Keep the station operational.
 department-Medical-description = Keep the crew healthy.
 department-Security-description = Keep the peace around the station.
 department-Science-description = Research new technologies and dangerous artifacts.
+department-Specific-description = Jobs that not all station have.
```
Comment (Emisse) [inline Resources/Locale/en-US/job/department-desc.ftl:8]:
```suggestion
department-Specific-description = Jobs that not all stations have.
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 2]
```diff
@@ -31,14 +31,14 @@ job-description-librarian = Manage the library, give out knowledge to any who se
 job-description-mime = Entertain the crew through non-vocal means, and engage with light rivalry with the clown.
 job-description-musician = Entertain the crew with your unique musical talent, and acquire new instruments to mess around with.
 job-description-passenger = Enjoy your stay aboard the station with no obligations!
-job-description-psychologist = Provide emotional support to traumatized crew. Not available on all maps!
+job-description-psychologist = Provide emotional support to traumatized crew. Currently available on Begel and Marathon Station.
```
Comment (Emisse) [inline Resources/Locale/en-US/job/job-description.ftl:34]:
```suggestion
job-description-psychologist = Provide emotional support to traumatized crew. Currently available on Marathon Station.
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 3]
```diff
@@ -31,14 +31,14 @@ job-description-librarian = Manage the library, give out knowledge to any who se
...
 job-description-qm = Manage the supplies of the station & the cargo department, keep the salvage specialists working, make sure all orders are fulfilled, and keep the money flowing.
 job-description-rd = Manage the science department, unlocking technologies, acquiring & researching artifacts, and performing experiments.
 job-description-research-assistant = Learn the basics of how to research various artifacts and anomalies.
-job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras and the radio. Not available on all maps!
+job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras and the radio. Currently available on Begel and Marathon Station.
```
Comment (Emisse) [inline Resources/Locale/en-US/job/job-description.ftl:38]:
```suggestion
job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras and the radio. Currently available on Bagel Station.
```

**Pull request merged**